checker/main.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2020 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/RedHatInsights/insights-content-service/groups"
+	"github.com/RedHatInsights/insights-results-aggregator/content"
+	"github.com/RedHatInsights/insights-results-aggregator/logger"
```

Comment:
A nitpick
I see that in Go code the imports are divided in 3 blocks: standard Golang library, external dependencies and current repository/project dependency.
Maybe the content and logger imports should be in a separated block from the groups

Reply:
Fair enough. I assume this separation was done automatically by `go fmt` or `goimports`.

Reply:
TBH I only follow the "best practices" that I see in our repositories, as a language newbie :-D

Reply:
@joselsegura I typically write our imports manually and/or by copy-pasting the existing ones. Otherwise I agree with what you said, of course. I noticed I didn't really do the separation right the first time, so now I hopefully finally fixed it.